cmd/kflex/common/cp.go

```diff
@@ -89,3 +91,29 @@ func convertToMap(pairs []string) (map[string]string, error) {
 
 	return params, nil
 }
+
+func (cp *CP) List(chattyStatus bool) {
+	clientset, err := client.GetClientSet(cp.Kubeconfig)
+	if err != nil {
+		fmt.Printf("Error getting clientset: %s\n", err)
+		os.Exit(1)
+	}
+
+	cps, err := clientset.CoreV1().CustomResourceDefinitions().List(cp.Ctx, metav1.ListOptions{
+		LabelSelector: "tenancy.kflex.kubestellar.io/controlplane",
+	})
+	if err != nil {
+		fmt.Printf("Error listing control planes: %s\n", err)
+		os.Exit(1)
+	}
+
+	if len(cps.Items) == 0 {
+		fmt.Println("No control planes found.")
+		return
+	}
+
+	fmt.Println("Control Planes:")
+	for _, cp := range cps.Items {
+		fmt.Printf("- %s\n", cp.Name)
```

Comment:
would you mind printing additional fields in addition to the name? Ideally this command should be on par with `kubect get cps` - example:

```console
$ k get cps
NAME   SYNCED   READY   TYPE   AGE
cp1    True     True    k8s    8m36s
```

Reply:
@asmit27rai could you please amend your commit for DCO signing ? See for example [here](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) for instructions.